config/crd/bases/infrastructure.cluster.x-k8s.io_awsclustertemplates.yaml

```
@@ -696,6 +701,7 @@ spec:
         type: object
     served: true
     storage: true
+    subresources: {}
```

Not sure how is it used, but in other templates I see it as below
```suggestion
   subresources:
      status: {}
```

---

Hi @Ankitasw, this is automatically generated file and cannot be modified.